src/comments/comments.service.ts

```diff
@@ -29,17 +29,9 @@ export class CommentsService {
     if (!result) {
       throw new Error('Could not add comment');
     }
-    console.log('test comment service');
 
-    console.log(result);
-
-    console.log('comment service post: ', postId);
-    console.log('comment service user: ', user);
-    console.log('comment service text: ', text);
-
-    await this.userService.addCommentToUser(user, result._id);
-    await this.postsService.addCommentToPost(postId, result._id);
-    // await this.postsService.updatePost(post, result.id);
+    await this.userService.addCommentToUser(user, result);
+    await this.postsService.addCommentToPost(postId, result);
 
     return result._id as string;
   }
```

Comment:
Return types for the rest of your methods.

Simplify stuff like:

```
    const comments = await this.commentModel.find().populate('user').exec();
    return comments;
```
to
`
   return await this.commentModel.find().populate('user').exec();`

Reply:
Also, we talked about this but for your findCOmment method, use the right exception. the error in the catch block is not a notfoundexception

Reply:
If the comment is not found in the DB, should that not be a NotFoundException? Perhaps I am assuming too much, and I'll look more into nest exceptions.